spec/unit/application/device_spec.rb

```diff
@@ -265,12 +271,12 @@
     end
   end
 
-
   describe "when running" do
     before :each do
       @device.options.stubs(:[]).with(:fingerprint).returns(false)
       Puppet.stubs(:notice)
       @device.options.stubs(:[]).with(:detailed_exitcodes).returns(false)
+      @device.options.stubs(:[]).with(:node).returns(false)
```

Comment:
Above you ensure that it is init'd to `nil`, but here you stub with `false`. Should this be `nil` instead?